worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/CreditStreamManager.java

```diff
@@ -252,6 +251,16 @@ public int getActiveMapPartitionCount() {
     return activeMapPartitions.size();
   }
 
+  public void cleanupExpiredShuffleKey(Set<String> expiredShuffleKeys) {
```

Comment:
This change could significantly delay the cleanup time of stream-related objects, as for Flink shuffle cleanup only occurs when app ends. This may result in a large streamState objects in memory. I suggest that the related app metrics can be recorded and cleaned up within StreamState.